### ISSUE TITLE
New project errors out when local Git is not configured with user's email address

### ISSUE THREAD

**Issue by JorgeRdg** [boostercloud/booster#637]
Hey team!

Wanted to give Booster a try in my laptop and, when trying to create a new project, it **errored out** because it couldn't find Git's email address from Git's config (indeed, it's not configured in my system). Error description:
```
⠋ Initializing git repositoryError: Error: Could not initialize git repository
Initialized empty Git repository in /home/mykra/code/booster/booster-demo/.git/

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'mykra@Lindelap.(none)')

```
Besides, the process **hangs** loading with the `Initializing Git repository` message. 

This is low priority because the message is really clear but just wanted to point out that this exists.

**Comment by Fecony** [comment]
Hey @JorgeRdg This issue was fixed in #968(#1031) PR. You can check to see if it is still relevant and close the issue if it isn't  :tada:

**Comment by AdrianLorenzoDev** [comment]
Since this was already solved, let's close this issue! Thank you @Fecony and @JorgeRdg for your help! ❤️
